packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -68,6 +77,17 @@ class Layout extends React.Component {
             />
           </nav>
           <main id="main-container" className={cx('main-container')}>
+            {siteAlert.map(
```

Comment:
this is still going to error out if siteAlert isn't required, and doesn't have a default props type of [] empty array.
we should probably have a test for this specifically with and without alerts to show both scenarios work

Reply:
If siteAlert is just a component prop type (and not required), rendering it will do nothing...

Reply:
we are calling the `.map` function on siteAlert. if its not required, and its not passed in, it will default to undefined, and throw a run time exception correct? If we make a default prop value of [] it won't. Am I missing something?

Reply:
I didn't pass in any value for siteAlert. It didn't throw any error in console.

Reply:
I passed empty array.

Reply:
not empty array. How about not passing siteAlert prop at all. You will get error. If there is no alert then you should not expect parent component to pass empty array. Agree with Alex. That is why I made this comment. Use conditional rendering, If it is undefined it will not throw alert component if it is undefined.